cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/templates/etl/common/Constants.java

```diff
@@ -44,4 +44,11 @@
   public static final class Transform {
     public static final String SPECIFICATIONS = "templates.etl.adapter.transform.specifications";
   }
+
+  /**
+   * Constants related to Realtime Adapter.
+   */
+  public static final class Realtime {
```

Comment:
It's easier to read as long as we don't create a lot of inner class for that. E.g.

```java
Constants.REAL_TIME_UNIQUE_ID
Constants.REAL_TIME_TIMEOUT
Constants.REAL_TIME_DELAY
```

vs

```java
Constants.RealTime.UNIQUE_ID
Constants.RealTime.TIMEOUT
Constants.RealTime.DELAY
```